kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -1,622 +1,3 @@
-
-// add some dummy features onto the Exercises object to make khan-exercises.js happy
```

Comment:
Bunch of models.js code that got remerged in here due to merge conflict from Nalanda-rct3.